packages/block-library/src/breadcrumbs/index.php

```diff
@@ -448,21 +448,69 @@ function block_core_breadcrumbs_get_terms_breadcrumbs( $post_id, $post_type ) {
 		return array();
 	}
 
-	// Find the first taxonomy that has terms assigned to this post.
+	/**
+	 * Filters the preferred taxonomy and term for breadcrumbs on a per-post-type basis.
```

Comment:
High-level question, and also a non-leading one: how does this hook-based solution weigh up against one relying on user-editable block attributes? I'm not implying that they are mutually exclusive, but this seems like useful justification for this PR.

Is this hook something requested or validated by precedents in [x3p0-breadcrumbs](https://github.com/x3p0-dev/x3p0-breadcrumbs)?

Reply:
Some plugins (Woo breadcrumbs) have filters and others a UI(or maybe both). As you said they are not mutually exclusive, but I think plugin authors would mostly use that filter during registration of post types/taxonomies. So I see value in having a filter.

Having said that, we can add the UI later on too.

Do you think we should only have a UI for these?